caffe2/CMakeLists.txt

```diff
@@ -442,6 +442,8 @@ if (NOT INTERN_BUILD_MOBILE)
     ${TORCH_SRC_DIR}/csrc/jit/fuser/executor.cpp
     ${TORCH_SRC_DIR}/csrc/jit/fuser/codegen.cpp
     ${TORCH_SRC_DIR}/csrc/jit/fuser/fallback.cpp
+    ${TORCH_SRC_DIR}/csrc/throughput_benchmark/init.cpp
```

Comment:
nit: PT folder structure is relatively flat, it feels like a wrong top-level directory. How about calling it csrc/profiling or csrc/perftools ?